File: PersonnelManagement/PersonnelManagement/Models/Personill.cs
using System;
using System.Collections.Generic;

namespace PersonnelManagement.Models
{
    public partial class Personill
    {
        public int Id { get; set; }
        public int Person { get; set; }
        public int Illtype { get; set; }
        public int Illcode { get; set; }
        public DateTime Datestart { get; set; }
        public DateTime Dateend { get; set; }
        public int Illregime { get; set; }
        public string Illwho { get; set; }
        public sbyte Privelege { get; set; }
    }
}

File: PersonnelManagement/PersonnelManagement/Front/classes/personill.ts
export default class Personill {
    id: number;
    person: number;
    illtype: number;
    illcode: number;
    datestart: Date;
    dateend: Date;
    illregime: number;
    illwho: string;
    privelege: number;

    datestartString: string;
    dateendString: string;
    illtypeBool: boolean;
    privelegeBool: boolean;

    displayPrivelege: boolean;
}